google/cloud/pubsublite/types/__init__.py

```diff
@@ -13,23 +13,26 @@
 # limitations under the License.
 #
 
+from google.cloud.pubsublite_v1.types.common import Reservation
```

Comment:
since "Topic" and "Subscription" aren't here, I don't think "Reservation" should be either?

Reply:
that's right. Removed.